credit_integration/models.py

```diff
@@ -1,10 +1,13 @@
 from auditlog.registry import auditlog
-from django.db import models
+from django.contrib.postgres.fields import JSONField
```

Comment:
Just a note that this field is deprecated in Django 3.1 and replaced with [django.db.models.JSONField](https://docs.djangoproject.com/en/3.2/ref/models/fields/#django.db.models.JSONField). The migration shouldn't be an issue when a Django update is relevant, though.